### PR TITLE
User Input Form for Simulation Page

### PR DESCRIPTION
**Pull request by jennygaoo**
See screenshot at bottom

### CODE REVIEW

**Comment by andreahu12** [comment]
Looks nice! Thanks for the screenshot.

File: postal-injection/src/main/webapp/data_display.html [hunk 1]
```diff
@@ -36,7 +36,39 @@
...
+        </p>
+        <p>
+          <label><b>Pallet Shipping Rate</b></label>
+          <input class="w3-input w3-border" name="pallet-rate" type="text" value="0.25" />
+        </p>
+      </div>
+      <div class="user_input_form_column">
+        <label><b>Post Office Flat Rate</b></label>
+        <p>
+          <input class="w3-input w3-border" name="PO-rate" type="text" value="2.50" />
```
Comment (jgao23) [inline postal-injection/src/main/webapp/data_display.html:58]: supernit: name="post-office-flat-rate"

File: postal-injection/src/main/webapp/data_display.html [hunk 2]
```diff
@@ -36,7 +36,39 @@
     <!-- End NavBar -->
 
     <!-- Start Form -->
-    <!-- TODO(jenny): complete form  -->
+    <form class="w3-container w3-card-4" id="user_input_form">
+      <h2>Enter Your Parameters</h2>
+      <div class="user_input_form_column">
+        <label><b>Single Zip Code</b></label>
```
Comment (jgao23) [inline postal-injection/src/main/webapp/data_display.html:42]: what if i want all zip codes?

Comment (jfewii) [reply postal-injection/src/main/webapp/data_display.html:42]: +1

Comment (jfewii) [reply postal-injection/src/main/webapp/data_display.html:42]: +1

**Comment by jennygaoo** [comment]
updated screenshot
Screenshot 2020-07-16 at 4 15 49 PM

**Pull request merged**